src/js/hca/containers/ConfirmationPage.jsx

```diff
@@ -52,7 +52,7 @@ export class ConfirmationPage extends React.Component {
           <h4 className="confirmation-guidance-heading">What happens after I apply?</h4>
           <p className="confirmation-guidance-message"><a href="/health-care/after-you-apply">Find out what happens after you apply.</a></p>
           <h4 className="confirmation-guidance-heading">Need help?</h4>
-          <p className="confirmation-guidance-message">If you have questions, call <a href="tel:+1-877-222-8387">1-877-222-VETS (8387)</a> and press 2, Monday - Friday, 8:00 a.m. - 7:00 p.m. (ET).</p>
+          <p className="confirmation-guidance-message">If you have questions, please call <a href="tel:+1-877-222-8387">1-877-222-VETS (8387)</a> and press 2, Monday - Friday, 8:00 a.m. - 7:00 p.m. (ET).</p>
```

Comment:
@peggygannon I noticed these additional changes you requested on the original issue, kindly find an updated screenshot for your reference below:
![localhost-3001-health-care-apply-application-confirmation 2](https://user-images.githubusercontent.com/16051172/31687209-6ec41bd2-b33d-11e7-8aae-fcb661a2b6b5.png)